add SonarQube-access meeting protocol (#18)
Gamify-IT/docs: protocols/devops/2022-06-01-protocol.md

# Meeting Transcript 01.06.2022
Accessing SonarQube

- present: Gilian, Timo
- absent excused:
- absent unexcused:

## How to access SonarQube

> - actual only within uni-vpn accessable
> - discord voice chat is not accessable within the uni-vpn

Reviewer: ```suggestion
- provided SonarQube instance is only accessable within uni network
- Discord voice chat is not accessable within the uni network
- VPN Setup takes a lot of time
```

> Result:
> 
> - it isn't practicable to access SonarQube always within the uni-vpn
> - cause of that we need make the SonarQube web interface accessable from the internet
> - therefor we can ask Prof. Becker to setup a reverse proxy
> - if this is not possible we maybe can set up a proxy at a student-hosted server

Reviewer: ```suggestion
## Result

- it isn't practicable to access SonarQube within the uni-vpn
- SonarQube web interface should accessable from the internet
- we can ask Prof. Becker to setup a reverse proxy
- if this is not possible, maybe we can set up a proxy at a student-hosted server
```
